### DISCUSSION TITLE
无法输入英文

### DISCUSSION THREAD

**Discussion by Initsnow** [tumuyan/rime-melt#4] [Q&A]
无论是融合拼音还是Easy English Nano都不行，融合拼音可以打中文，Easy English Nano什么都打不了

**Comment by renzhengzhang** [comment]
使用融合拼音处于中文时输入 "hello"

![Snipaste_2022-12-01_00-27-42](https://user-images.githubusercontent.com/24836504/204854524-72825bb7-088c-4629-9bd9-41179add6eca.png)

**Reply by Initsnow** [reply]
在melt_pinyin.dict.yaml里的import_tables:添加  - melt_eng一行就可以了。
但这样还用lua、滤镜什么的干嘛😓
用来用去还是感觉没微软拼音好用，无论美观程度还是兼容程序方面
